config/config.php

```diff
@@ -18,22 +18,24 @@
                 'dsn' => env('SENTRY_LARAVEL_DSN', env('SENTRY_DSN')),
 
                 // capture release as git sha
-                // 'release' => trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD')),
+		'release' => env('SENTRY_RELEASE_GIT', false)
+			? trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD'))
```

Comment:
Nah, not going to be executing console commands in config files, that's just a bad idea.